shared/src/packages/stripe/libs/types/get-payments-request.type.ts

```diff
@@ -0,0 +1,16 @@
+import { type BusinessEntityT } from '~/packages/business/business.js';
+import { type OrderEntity } from '~/packages/orders/orders.js';
+
+type GetPaymentsRequest = {
+  businessId?: BusinessEntityT['id'];
+  orderId?: OrderEntity['id'];
+  userId?: OrderEntity['userId'];
+  customerName?: OrderEntity['customerName'];
+  customerPhone?: OrderEntity['customerPhone'];
+  intervalFrom?: Date;
+  intervalTo?: Date;
```

Comment:
I have a dedicated converter in joi validator, that converts it into date. Evenually this will be used to convert it to unixSeconds anyway.
see shared\src\packages\stripe\libs\validation-schemas\get-payments.validation-schema.ts